apis/v1alpha1/tlsroute_types.go

```diff
@@ -22,7 +22,6 @@ import (
 // TLSRouteSpec defines the desired state of TLSRoute
 type TLSRouteSpec struct {
 	// Rules are a list of TLS matchers and actions.
-	// +required
 	// +kubebuilder:validation:MinItems=1
 	Rules []TLSRouteRule `json:"rules"`
```

Comment:
Add a max limit of 16 like we have done in HTTPRoute.

Reply:
Updated, thanks!